build/release-notes/avalanchego.md

```diff
@@ -2,6 +2,26 @@
 
 {% page-ref page="../tutorials/nodes-and-staking/upgrade-your-avalanchego-node.md" %}
 
+## PRE\_RELEASE v1.6.0-fuji \([View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.6.0-fuji)\)
+
+**Please note that this release is unable to run mainnet - and will display "this node version doesn't support mainnet" if attempted to run with a mainnet configuration. If you run a mainnet node, no action is required until the official release is published soon.**
+
+**All Fuji nodes should upgrade before 5 PM EDT, September 16th 2021.**
```

Comment:
I put this below the `The changes in the upgrade go into effect at 5 PM EDT...` line in the release notes